.github/workflows/monthly.yml

```diff
@@ -0,0 +1,41 @@
+name: monthly
+
+on:
+  push
+    branches:
+      - master
```

Comment:
I think this should be removed, we only want the scheduled part.

Reply:
Ooh, okay. I am not sure about this but since you're way more experienced, I'll drop this :smile: